linker/shared/src/main/scala/org/scalajs/linker/analyzer/Analyzer.scala

```diff
@@ -427,19 +432,26 @@ private final class Analyzer(config: CommonPhaseConfig,
     var instantiatedSubclasses: List[ClassInfo] = Nil
     var methodsCalledLog: List[(String, From)] = Nil
 
-    lazy val (methodInfos, staticMethodInfos) = {
-      val allInfos = for (methodData <- data.methods)
-        yield (methodData.encodedName, new MethodInfo(this, methodData))
-      val (staticMethodInfos, methodInfos) = allInfos.partition(_._2.isStatic)
-      (mutable.Map(methodInfos: _*), mutable.Map(staticMethodInfos: _*))
+    private lazy val nsMethodInfos = {
```

Comment:
Seems kind of pointless to make this a lazy val if you evaluate it anyways for `publicMethodInfos` below. (this is a bump).